app/main.py

```diff
@@ -1,13 +1,38 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [{
+            "line": every["line_number"],
+            "column": every["column_number"],
+            "message": every["text"],
+            "name": every["code"],
+            "source": "flake8"
+        } for every in errors if every["filename"] == file_path],
+        "path": file_path,
+        "status": "failed" if len(errors) > 0 else "passed"
```

Comment:
There is no need to evaluate the length of the list, an empty list is faulty by default. Rewrite this part to encounter that.

Reply:
If i call this function in next function, i must have this status F or T.

Reply:
There is no need to evaluate the length, it will not change the logic. Change the code to the following:
```suggestion
        "status": "failed" if errors else "passed"
```